modules/ProjectScore.py

```diff
@@ -0,0 +1,186 @@
+"""Disease to target evidence parser for Project Score v2."""
+import argparse
+import logging
+
+from pyspark.sql import DataFrame, SparkSession
+from pyspark.sql import functions as f
+from pyspark.sql import types as t
+
+from common.evidence import (
+    GenerateDiseaseCellLines,
+    initialize_logger,
+    initialize_sparksession,
+    write_evidence_strings,
+)
+
+# This parser is specific for the second release of Porject Score, so the publication identifier is hardcoded
+PMID = "38215750"
```

Comment:
This is inside `generate_project_score_evidence`
```suggestion
```

Reply:
Oh, yes.